.ci/jenkins/dsl/jobs.groovy

```diff
@@ -19,15 +20,52 @@ Map getMultijobPRConfig() {
                 primary: true,
                 // TODO remove once https://issues.redhat.com/browse/KOGITO-4113 is done 
                 // as it will become the default path
-                jenkinsfile: OPTAPLANNER_JENKINSFILE_PATH
+                jenkinsfile: BUILD_CHAIN_JENKINS_PATH,
+                git: [
```

Comment:
this will not work like that. That information could be given in `getDefaultJobParams` which uses https://github.com/kiegroup/kogito-pipelines/blob/master/dsl/seed/src/main/groovy/org/kie/jenkins/jobdsl/templates/KogitoJobTemplate.groovy#L440

btw, why do we need that URL ?

Reply:
edit: `jenkinsfile` and `git` information should be removed as they are useless here.
They will be set automatically to correct value by the template method due to `buildchain: true`, aka `Jenkinsfile.buildchain` and `kogito-pipelines` repository url